keps/sig-api-machinery/2887-openapi-enum-types/README.md

```diff
@@ -580,13 +581,17 @@ feature flags will be enabled on some API servers and not others during the
 rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
+If the API-server has multiple replicas, with some instances not yet enabling this feature,
+whether the returned OpenAPI Spec contains enum types will depend on which instance holds the leader lock. 
```

Comment:
what leader lock?

Reply:
Oops, I've been working too much with leader election recently. I meant depends on which instance handled the request.

Reply:
Updated.